draft.md

```diff
@@ -65,7 +68,8 @@ Release Date: 2019-02-11
 
 ###  Tutorials
 
-
++ [Enhancing gather() and spread() by Using "Bundled" data.frames](https://yutani.rbind.io/post/enhancing-gather-and-spread-by-using-bundled-data-frames/)
```

Comment:
This was already included in last weeks edition of RWeekly.

Reply:
oops! And I was the one that added it last week too, sorry.  I deleted it and added a different thing instead.